src/auto_pickup.cpp

```diff
@@ -343,9 +345,9 @@ void auto_pickup::show( const std::string &custom_name, bool is_autopickup )
             if( g->u.name != "" ) {
                 save_character();
             }
-        } else {
-            create_rules();
         }
+
+        ready = false;
```

Comment:
I've seen it too, but you could have got rid of it completely. Private things don't require doxygen comments, clearer names are preferable.